nixos/modules/programs/xppen.nix

```
@@ -0,0 +1,44 @@
+{ config, lib, pkgs, ... }:
+
+with lib; let
+  cfg = config.programs.xppen;
+in
+
+{
+  options.programs.xppen = {
+    enable = mkEnableOption (lib.mdDoc "XPPen PenTablet application");
```

```suggestion
    enable = mkEnableOption "XPPen PenTablet application";
```
`lib.mdDoc` is now just an alias and can be safely removed everywhere.
see https://github.com/NixOS/nixpkgs/commit/d36f950d405369abca5144b14a201bc25c4a080a and https://github.com/NixOS/nixpkgs/pull/237557